server/config/config.go

```diff
@@ -829,10 +826,12 @@ type SchedulerConfigs []SchedulerConfig
 
 // SchedulerConfig is customized scheduler configuration
 type SchedulerConfig struct {
-	Type        string   `toml:"type" json:"type"`
-	Args        []string `toml:"args" json:"args"`
-	Disable     bool     `toml:"disable" json:"disable"`
-	ArgsPayload string   `toml:"args-payload" json:"args-payload"`
+	Type        string                 `toml:"type" json:"type"`
+	Args        []string               `toml:"args" json:"args"`
+	Disable     bool                   `toml:"disable" json:"disable"`
+	ArgsPayload string                 `toml:"args-payload" json:"args-payload"`
+	Name        string                 `toml:"name" json:"name"`
+	Payload     map[string]interface{} `toml:"payload" json:"payload"`
```

Comment:
It's better to remove in `SchedulerConfig` because each scheduler has its configuration now, we may remove global `SchedulerConfig` in the feature.

Reply:
Do you mean deleting the schedulers-v2 and schedulers-payload from "pd-ctl config show" output, and add these information to "pd-ctl scheduler config show" command?

Reply:
no, I mean we only need `schedulers-payload`,  and hide ` schedulers-v2 ` in "pd-ctl config show".